### ISSUE TITLE
Add NodeJS Learning Resource

### ISSUE THREAD

**Issue by Bachittarjeet** [UNO-Coders/Web-Development#42]

**Comment by MuhammadTanveer94** [comment]
@Bachittarjeet could you please assign me?

**Comment by Bachittarjeet** [comment]
Sure @MuhammadTanveer94

**Comment by MuhammadTanveer94** [comment]
@Bachittarjeet could you please describe? what i have to do for this task?

**Comment by Bachittarjeet** [comment]
Yes. you can add different types of code snippets like simple login form, basic file handling in node, or maybe using helmet or passport.js integration. basically you can add different code snippets which are beginner friendly. I hope this helps :)

**Comment by jyshdewangan** [comment]
Hey, @Bachittarjeet can you please assign this to me? If it is still inactive

**Comment by NazimFilzer** [comment]
@Bachittarjeet  Is this Still Open??

**Comment by Bachittarjeet** [comment]
yes @NazimFilzer

**Comment by nikhil-7span** [comment]
@Bachittarjeet Can you please assign me above issue. I will do that and I am sharing some references resource for NodeJS.

Thanks!
